### DISCUSSION TITLE
Feature request: option to fold (hide) output

### DISCUSSION THREAD

**Discussion by lbenet** [quarto-dev/quarto-cli#4911] [Feature Requests]
Among other, I use quarto for teaching mostly with html (but not exclusively). Often, I ask the students to work on an exercise, which involves writing some code and obtain results, and after few minutes, I solve the exercise and then we discuss the results; I think this is pretty standard. Folding the code is clearly very useful; yet, it would be also nice to have the results folded/hidden, just as the code is.

**Comment by cderv** [comment]
For reference, this is tracked in https://github.com/quarto-dev/quarto-cli/issues/341

**Comment by lbenet** [comment]
Closing, in favor of following the discussion in #341
